.github/workflows/CI.yml

```diff
@@ -116,3 +116,54 @@ jobs:
       - name: Test
         run: |
           ${{ matrix.config.qemu }} build/uv_run_tests_a
+
+  build-cross-mingw:
+    runs-on: ubuntu-latest
+    name: build-cross-mingw
+    steps:
+      - uses: actions/checkout@v2
+      - name: Envinfo
+        run: npx envinfo
+      - name: Install mingw32 environment
+        run: |
+          sudo apt update
+          sudo apt install mingw-w64 -y
+      - name: Build
+        run: |
+          mkdir build
+          cd build && cmake .. -DBUILD_TESTING=ON -DQEMU=ON -DCMAKE_TOOLCHAIN_FILE=../cmake-toolchains/cross-mingw32-linux.cmake
+          cmake --build .
+          ls -lh
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: Artifacts
+          path: build/**/*
+
+  test-mingw:
+    runs-on: windows-${{ matrix.config.server }}
+    name: test-mingw-${{ matrix.config.arch}}
+    needs: build-cross-mingw
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {arch: Win32, server: 2022}
+          - {arch: x64, server: 2022}
```

Comment:
This is just running the i686 tests twice, since that is the only binary that we built. Can we make the build step have a toolchain for both i686 and x86_64 in separate build folders?

Reply:
That's... actually correct. I lost track of this fact. I can try doing the 64 bits build too.